src/Common/tests/System.Net/Sockets/SocketTestServerAsync.cs

```diff
@@ -220,38 +220,36 @@ private void IO_Completed(object sender, SocketAsyncEventArgs e)
             switch (e.LastOperation)
             {
                 case SocketAsyncOperation.Receive:
-                ProcessReceive(e);
-                break;
+                    ProcessReceive(e);
```

Comment:
> Are you using the corefx VS settings?

This is what the formatter gave me, yes.

> as long as these are spaces and not tabs :)

Hah, yes--they are spaces this time around.